integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/chat_generator.py

```diff
@@ -208,9 +208,18 @@ def run(
 
         # rename the meta key to be inline with OpenAI meta output keys
         for response in replies:
-            if response.meta is not None and "usage" in response.meta:
-                response.meta["usage"]["prompt_tokens"] = response.meta["usage"].pop("input_tokens")
-                response.meta["usage"]["completion_tokens"] = response.meta["usage"].pop("output_tokens")
+            if response.meta:
+                if "usage" in response.meta:
+                    if "input_tokens" in response.meta["usage"]:
+                        response.meta["usage"]["prompt_tokens"] = response.meta["usage"].pop("input_tokens")
+                    if "output_tokens" in response.meta["usage"]:
+                        response.meta["usage"]["completion_tokens"] = response.meta["usage"].pop("output_tokens")
+                else:
+                    if "prompt_token_count" in response.meta:
+                        response.meta["usage"] = {}
```

Comment:
```suggestion
                    response.meta["usage"] = {}
                    if "prompt_token_count" in response.meta:
```

maybe it is safer to do that